src/lib/storage/storage_manager.hpp

```diff
@@ -60,16 +87,66 @@ class StorageManager : public Noncopyable {
   // For debugging purposes mostly, dump all tables as csv
   void export_all_tables_as_csv(const std::string& path);
 
+  void persist_chunks_to_disk(const std::vector<std::shared_ptr<Chunk>>& chunks, const std::string& file_name);
+
+  // These functions are for the moment public, to test them properly.
```

Comment:
why do they need to be public to be tested?

Reply:
this comment is a residue, the remaining public funtions where meant to be part of the public interface as far as I understand

Reply:
Comment will be deleted